float32 test_time
int32 num_joints
int32 num_actuators
JointData[] joint_data
ActuatorData[] actuator_data
---
